spec/models/book_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :title }
+    it { should validate_presence_of :description }
+    it { should validate_presence_of :image }
+    it { should validate_presence_of :est_reading_time }
+    it { should validate_presence_of :page_count} 
+  end
+
+  describe 'relationships' do
+  end
+
```

Comment:
Just brainstorming here for when we update this: 
book can have many authors/images
book can have one title/description
book belongs to a category
Thoughts?